### PR TITLE
fix: tap event was not sending the right handle to start()

### PR DESCRIPTION
**Pull request by vvo**

### CODE REVIEW

File: src/js/scope_events.js [hunk 1]
```diff
@@ -195,7 +195,7 @@
 		fireEvent('change', handleNumber);
 
 		if ( options.events.snap ) {
-			start(event, { handles: [scope_Handles[total]] });
+			start(event, { handles: [scope_Handles[handleNumber]] });
```
Comment (leongersen) [inline src/js/scope_events.js]: Is there a test for this? I haven't found the bug this fixes yet.

Comment (vvo) [inline src/js/scope_events.js]: No there's no added test, it's only when you have snap option I guess.

Comment (vvo) [inline src/js/scope_events.js]: here: http://refreshless.com/nouislider/behaviour-option/#section-snap click on the bar, console throw

Comment (leongersen) [inline src/js/scope_events.js]: Right, got it now. [Merge progress](https://github.com/leongersen/noUiSlider/tree/merge).

Comment (vvo) [inline src/js/scope_events.js]: Awesome work you are filling the gaps in my PRs, hope it's ok. We will make an awesome module! Also I have this already working: https://github.com/algolia/react-nouislider

**Pull request merged**